iexfinance/base.py

```diff
@@ -83,6 +83,9 @@ def __init__(self, **kwargs):
                 "through the environmental variable "
                 "IEX_TOKEN."
             )
+        if self.token.startswith(("Tsk", "Tpk")):
+            logger.info("Using a sandboxed environment because a test key was detected")
+            os.environ["IEX_API_VERSION"] = "iexcloud-sandbox"
```

Comment:
This can just be ``sandbox``. Trying to phase out ``iexcloud-sandbox`` which was needed to distinguish from the legacy IEX API.

Reply:
Done.